src/utils/apis/TasksApi.js

```diff
@@ -0,0 +1,22 @@
+import {recruitmentServiceBasicAPILink} from "./APILinks";
+import Swal from 'sweetalert2'
+
+export const tasksApi = {
+
+    getTask: (processId) => {
+        if(processId){
+            return fetch(`${recruitmentServiceBasicAPILink}/api/tasks?process=${processId}`)
+                .then(response => response.json())
+        } else {
+            Swal.fire({
+                title: 'Oops!',
+                icon: 'error',
+                text: `Looks like somebody didn't add a task but wants to solve one something!`
```

Comment:
można to w apostrofy, nie ma tu żadnej interpolacji

Reply:
Niby tak, ale staram się używać ' zamiast " (niby bardziej jsowo), a jak było don't, to się wywalało. Jest jakiś znaczny spadek performance'u jeśli dam ` zamiast '

Reply:
Tu poszła w sumie czysta oszczędność czasu. ' nie może być, bo jest didn't i się string na tym kończy. Do " trzeba wcisnąć shift, a to już było za dużo na moje nerwy. A ` można sobie na spokojnie trafić nie patrząc XDDD